packages/components/src/KaizenProvider/ThemeProvider/themes/types.ts

```diff
@@ -0,0 +1,250 @@
+import type { CSSProperties as ReactCSSProperties } from "react"
+
+/**
+ * Use a custom type for CSS properties becauase we should eventually write a more strongly typed version using template literal types.
+ * This could definitely be contributed back to the community too. An example starting point here https://github.com/ghoullier/awesome-template-literal-types#css-parser
+ * For example:
+ * ```ts
+ * type Font = {
+ *   fontSize: `${number}rem`,
+ *
+ *  }
+ * type HexDigit = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | A | B | C | D | E | F ;
+ * type Color = `rgba(${number}, ${number}, ${number}, ${number}) | #${HexDigit}${HexDigit}${HexDigit}` // You get the point
+ *
+ * ```
+ * */
+type KaizenCSSProperties = ReactCSSProperties
+
+type Hex = string
+// Once TypeScript is upgraded in the repo, you can use Lowercase<>
```

Comment:
Has TS been upgraded yet? 😂

Reply:
Lol touching this broke everything, maybe later :P